### PR TITLE
add html content and modify it using css to add toolbar and headline section

### PR DESCRIPTION
**Pull request by Gedewon**
### In this milestone, I:
- Built the toolbar and headline section
- Used linters for code standard
- Used flex to parse Figma layout

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,10 +4,58 @@
...
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Portifolio</title>
-    <link rel="stylesheet" href="styles/styles.css" />
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="/styles/styles.css" />
   </head>
   <body>
-    <h1>microverse portifolio</h1>
+    <header>
```
Comment (BenMukebo) [inline index.html:11]: - [ ] Great work up to here, you have implemented HTML best practice, I salute your effort, I think now working on adding some improvement in the styling is crucial, see the image below and compare to the Figma design, please adjust your styling (`padding, colors, margins, font-weight, font-size, background`) to match as much as possible the original design
- [ ] Your design
gede

- [ ] Figma Design
figgede

**Review by BenMukebo — CHANGES_REQUESTED** [review]
### Hi **@gedewon**

You did a great job so far 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!

### To highlight 💪🏻

- No linter errors. ✔️ 
- Github flow was correctly used.  ✔️   
- Good commit messages ✔️ 
- Professional README ✔️ 
- Descriptive PR.  ✔️ 
 👍  Good job!


## STATUS: CHANGES REQUESTED  ♻️ 
_Please Check the comments under the review and make all required changes_

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏 💻  

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @BenMukebo ) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (prolajumokeoni) [reply index.html]: -  Good job @Gedewon kindly make the header look like the original, so far you are on track 👏🏿 👏🏿 

##OPTIONAL


Your design             |  Figma Design
:-------------------------:|:-------------------------:
bbbbbw  | aaaaw

- [ ] Kindly remove the border-bottom, change background colour and reduce padding to your design check screenshot above

Comment (Gedewon) [reply index.html]: @prolajumokeoni sure ,i am doing housekeeping for the Desktop version

**Review by prolajumokeoni — APPROVED** [review]
Hi @Gedewon 👋🏿 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
_Check inline comments_
<br>
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification  @prolajumokeoni.
153888587-3e47b8be-12ea-47db-a3c2-33122e091533

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (prolajumokeoni) [reply index.html]: - Ok great, keep rocking 💪🏿

**Pull request merged**